<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContentWidget</class>
 <widget class="QWidget" name="ContentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>t:slim X2</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Current Glucose</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>123 mg/dL</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Insulin on Board (IOB)</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="iobValue">
          <property name="text">
           <string>1.75 U</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="activeTimeLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Active Time:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="activeTime">
          <property name="text">
           <string>2:45 hr</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="bolusButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 10pt &quot;DejaVu Sans&quot;;</string>
         </property>
         <property name="text">
          <string>BOLUS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelBolus">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="optionsButton">
         <property name="minimumSize">
          <size>
           <width>256</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>OPTIONS</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <property name="minimumSize">
         <size>
          <width>778</width>
          <height>150</height>
         </size>
        </property>
        <attribute name="title">
         <string>History</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTableWidget" name="historyTable">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectItems</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string>Date/Time</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Type</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Details</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Graph</string>
        </attribute>
        <widget class="QCustomPlot" name="graph" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>781</width>
           <height>291</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Alerts</string>
        </attribute>
        <widget class="QTableWidget" name="alertsTable">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>30</y>
           <width>461</width>
           <height>271</height>
          </rect>
         </property>
         <property name="gridStyle">
          <enum>Qt::NoPen</enum>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Alerts</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>timestamp</string>
          </property>
         </column>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Settings</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QGroupBox" name="active_profile_groupbox">
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Active Profile Stats</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="setting_name_label">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Name:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="setting_br_label">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::NoContextMenu</enum>
              </property>
              <property name="text">
               <string>Basal Rate:</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="setting_cr_label">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Carb Ratio:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="setting_target_label">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Target Range&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="setting_cf_label">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Correction Factor:</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Settings</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_16">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;PIN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QSpinBox" name="setting_pin_1">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>9</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="setting_pin_2">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>9</number>
                </property>
                <property name="singleStep">
                 <number>1</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="setting_pin_3">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>9</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="setting_pin_4">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>9</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="setting_pin_update_button">
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="profile_tab">
        <attribute name="title">
         <string>Profiles</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QGroupBox" name="active_profile_tab">
           <property name="title">
            <string>Profile View</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QListWidget" name="profile_list"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QPushButton" name="profile_select_button">
                <property name="text">
                 <string>Select</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="profile_delete_button">
                <property name="text">
                 <string>Delete</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="edit_profile_tab">
           <property name="title">
            <string>Edit Existing Profile</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label_13">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Name</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPlainTextEdit" name="edit_profile_name_box">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="edit_profile_setting_box">
                <item>
                 <property name="text">
                  <string>BR</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>CR</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>CF</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Target Min</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Target Max</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_15">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>New Value</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="edit_val_box"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_14">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Setting</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="profile_edit_button">
              <property name="text">
               <string>Confirm</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="new_profile_tab">
           <property name="title">
            <string>Create New Profile</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_8">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Name</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPlainTextEdit" name="new_profile_name_box">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_9">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>BR</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="new_profile_br_box"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_10">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>CR</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="new_profile_cr_box"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_11">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>CF</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QDoubleSpinBox" name="new_profile_cf_box"/>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_12">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Target (Min, Max)</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QDoubleSpinBox" name="new_profile_max_box">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="new_profile_min_box">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="profile_create_button">
              <property name="text">
               <string>Create</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>headers/QCustomPlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
